src/components/v5/shared/ActionTypeNotification/ActionTypeNotification.tsx

```diff
@@ -0,0 +1,129 @@
+import { Extension } from '@colony/colony-js';
+import React, { FC } from 'react';
+import { defineMessages } from 'react-intl';
+import { useNavigate } from 'react-router-dom';
+
+import { ACTION } from '~constants/actions';
+import { useActionSidebarContext } from '~context';
+import { useColonyContext } from '~hooks';
+import { formatText } from '~utils/intl';
+import { useIsFieldDisabled } from '~v5/common/ActionSidebar/partials/hooks';
+import NotificationBanner from '~v5/shared/NotificationBanner';
+
+import { ActionTypeNotificationProps } from './types';
+
+const displayName = 'v5.ActionTypeNotification';
+
+const MSG = defineMessages({
+  unlockedToken: {
+    id: `${displayName}.unlockedToken`,
+    defaultMessage: `This Colony's native token has already been unlocked and can be traded.`,
+  },
+  unlockedTokenError: {
+    id: `${displayName}.unlockedTokenError`,
+    defaultMessage: 'This action is irreversible. Use with caution.',
+  },
+  enterRecoveryModeError: {
+    id: `${displayName}.enterRecoveryModeError`,
+    defaultMessage:
+      'This will disable the Colony and prevent any further activity until resolved.',
+  },
+  createDecisionError: {
+    id: `${displayName}.createDecisionError`,
+    defaultMessage:
+      'Agreements requires the Reputation weighted extension to be enabled.',
+  },
+  learnMore: {
+    id: `${displayName}.learnMore`,
+    defaultMessage: 'Learn more',
+  },
+  viewExtension: {
+    id: `${displayName}.viewExtension`,
+    defaultMessage: 'View extension',
+  },
+});
+
+export const ActionTypeNotification: FC<ActionTypeNotificationProps> = ({
+  selectedAction,
+  className,
+}) => {
+  const { colony } = useColonyContext();
+  const isNativeTokenUnlocked = !!colony?.status?.nativeToken?.unlocked;
+  const isFieldDisabled = useIsFieldDisabled();
+  const navigate = useNavigate();
+  const {
+    actionSidebarToggle: [, { toggleOff: toggleActionSidebarOff }],
+  } = useActionSidebarContext();
+
+  const actionTypeNotificationHref =
+    (selectedAction === ACTION.UNLOCK_TOKEN &&
+      'https://docs.colony.io/use/managing-funds/unlock-token/') ||
+    undefined;
+
+  const actionTypeNotificationTitle = (): string | undefined => {
```

Comment:
I think this name doesn't reflect that this is a function. Perhaps something like `getNotificationTitle` would be better